Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture.yml

```diff
@@ -0,0 +1,186 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: Recorded Future Feed
+  version: -1
+configuration:
+- defaultvalue: A - Completely reliable
+  display: Source reliability
+  name: feedInstanceReliability
+  options:
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  required: true
+  type: 15
+- name: expiration
+  required: false
+  type: 17
+- defaultvalue: suddenDeath
+  hidden: true
+  name: expirationPolicy
+  required: false
+  type: 0
+- hidden: true
+  name: expirationInterval
+  required: false
+  type: 1
+- display: Fetch indicators
+  name: feed
+  required: false
+  type: 8
+- additionalinfo: Indicators from this integration instance will be marked with this
+    reputation.
+  defaultvalue: feedInstanceReputationNotSet
+  display: Indicator reputation
+  name: feedInstanceReputation
+  options:
+  - None
+  - Good
+  - Suspicious
+  - Bad
+  required: false
+  type: 18
+- defaultvalue: '60'
+  display: Fetch Interval
+  name: feedFetchInterval
+  required: true
+  type: 19
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- additionalinfo: Type of the indicator in the feed.
+  display: Indicator type
+  hidden: false
+  name: indicator_type
+  options:
+  - domain
+  - ip
+  - hash
+  - url
+  required: true
+  type: 15
+- display: API token
+  hidden: false
+  name: api_token
+  required: true
+  type: 4
+- defaultvalue: connectApi
+  display: Sub Feeds
+  hidden: false
+  name: sub_feeds
+  options:
+  - connectApi
+  - fusion
+  required: true
+  type: 16
+- additionalinfo: |-
+    Limit the indicators list to a specific risk rule.
+    To see available risk rules run the rf-feed-get-risk-rules command.
+    This parameter will only be used for the 'connectApi' sub-feed.
+  display: Risk Rule
+  hidden: false
+  name: risk_rule
+  required: false
+  type: 0
+- additionalinfo: |-
+    Load a custom risklist from a specified Recorded Future file path.
+    If no file path is specified, the default risklist file is used.
+    This parameter will only be used for the 'fusion' sub-feed.
+  display: Fusion File Path
+  hidden: false
+  name: fusion_file_path
+  required: false
+  type: 0
+- additionalinfo: Time in seconds before HTTP requests timeout.
+  defaultvalue: '20'
+  display: Request Timeout
+  hidden: false
+  name: polling_timeout
+  required: true
+  type: 0
+- additionalinfo: The minimum score from the feed in order to to determine whether
+    the indicator is malicious. Default is "65"
```

Comment:
Perhaps include here a sentence or two about the scoring that Recorded Future uses as you showed us with your screenshot. Something along the lines of `Recorded Future's scoring system is out of 100. By default, Recorded Future deems indicators in the range blah blah blah to be Suspicious and blah blah blah Malicious etc.`.